docs/tutorial.md

```diff
@@ -344,6 +344,45 @@ Click **View Logs** for one of the samples to see the log messages that match th
 
 You can expand any of the messages that matches the filter to see the full stack trace.
 
+## SRE Recipes
+
+SRE Recipes is our [Chaos Engineering](https://en.wikipedia.org/wiki/Chaos_engineering) tool to test your sandbox environment. It helps users to familiarize with finding the root cause of a breakage using Cloud Ops monitoring.  
```

Comment:
nit: familiarize themselves

Reply:
nit: replace "using Cloud Ops monitoring' with "using Cloud Operations suite of tools" ( it's not only monitoring)